packages/lib/markdownUtils.ts

```diff
@@ -61,9 +61,10 @@ const markdownUtils = {
 		return url;
 	},
 
-	prependBaseUrl(md: string, baseUrl: string) {
+	prependBaseUrl(md: string, baseUrl: string, test?: Function) {
```

Comment:
Hopefully you can remove this `test` argument once you've updated HtmlToMd. I'm not keen on all these optional arguments being added to functions.